apps/sequence/values/values_controller.h

```diff
@@ -14,12 +14,17 @@ class ValuesController : public Shared::ValuesController {
   Button * buttonAtIndex(int index, ButtonRowController::Position position) const override {
     return const_cast<Button *>(&m_setIntervalButton);
   }
+  KDCoordinate columnWidth(int i) override;
+  KDCoordinate cumulatedWidthFromIndex(int i) override;
+  int indexFromCumulatedWidth(KDCoordinate offsetX) override;
   void willDisplayCellAtLocation(HighlightCell * cell, int i, int j) override;
   I18n::Message emptyMessage() override;
   IntervalParameterController * intervalParameterController() override {
     return &m_intervalParameterController;
   }
 private:
+  static constexpr KDCoordinate k_abscissaCellWidth = 100;
```

Comment:
Could be factorised with k_ordinateCellWidth and Graph::ValuesController:: k_cellWidth in Shared::ValuesController
columnWidth and indexFromCumulatedWidth become simpler?

Reply:
It was done later.